tests/integration/test_integration.py

```diff
@@ -717,7 +723,6 @@ def test_partial_flush_log(run_python_code_in_subprocess, encoding, monkeypatch)
     t = Tracer()
 
     t.configure(
-        partial_flush_enabled=True,
```

Comment:
same here, we might want to make sure we are testing the default behavior and with it disabled.

Reply:
Just needs the alternate test case with partial flushing disabled 👍

Reply:
Oh I think those were already added here: 
https://github.com/DataDog/dd-trace-py/blob/c317bcf5fd431289fc925f6580e9331f524c69b2/tests/integration/test_integration.py#L256-L288 

and here: 
https://github.com/DataDog/dd-trace-py/blob/c317bcf5fd431289fc925f6580e9331f524c69b2/tests/integration/test_integration.py#L317-L333 

Was there another test I missed?

Reply:
I think Brett might be referring to this specific test `test_partial_flush_log` so that we have a variant test case `test_partial_flushing_disabled_log` with partial flushing disabled and see the expected result in the log. Can you confirm @brettlangdon ?

Reply:
🤔 actually, re-reviewing the test, it might be ok to not have a with partial flushing disabled version, it doesn't hurt, but think this is "ok"